### PR TITLE
feat(game): add tooltip to display rounds left

### PR DESCRIPTION
**Pull request by RomainGuarinoni**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -28,7 +28,12 @@
 
     <div class="button-container">
       <button id="restart">Nouvelle partie</button>
-      <button id="verify">Vérifier</button>
+      <div class="verify-container">
+        <button id="verify">Vérifier</button>
+        <div id="tooltip">
+          <p>Il vous reste <span>11</span> tours</p>
```
Comment (Guillaume-FAURE) [inline public/index.html:34]: Why 11? the number of turn must have the possibility to be chosen

File: src/dom-manipulation.ts [hunk 2]
```diff
@@ -122,3 +122,30 @@ export function getGameDomElements(round: number) {
...
+  const span = tooltip.querySelector('span');
+
+  if (!span) {
+    throw new Error('No span found in the tooltip');
+  }
+
+  span.innerHTML = `${roundLeft}`;
+
+  if (roundLeft <= 4) {
+    span.style.color = 'red';
```
Comment (Guillaume-FAURE) [inline src/dom-manipulation.ts:141]: Maybe adapt it to be proportional to the number of turn max (if we begin at 7 turns max and it's already orange, it's strange

File: src/dom-manipulation.ts [hunk 3]
```diff
@@ -122,3 +122,30 @@ export function getGameDomElements(round: number) {
...
+    return;
+  }
+
+  if (roundLeft <= 8) {
+    span.style.color = 'orange';
```
Comment (Guillaume-FAURE) [inline src/dom-manipulation.ts:146]: same thing

**Review by Guillaume-FAURE — COMMENTED** [review]
Maybe some things to change/update but we can discuss it if you want.
No overall errors

File: src/dom-manipulation.ts (hunk 2 above)
Comment (RomainGuarinoni) [reply src/dom-manipulation.ts:155]: Done

File: src/dom-manipulation.ts (hunk 3 above)
Comment (RomainGuarinoni) [reply src/dom-manipulation.ts:160]: Done

File: public/index.html (hunk 1 above)
Comment (Guillaume-FAURE) [reply public/index.html:34]: true

**Review by Guillaume-FAURE — APPROVED** [review]
perfect

**Pull request merged**